dockerfiles/theia/build.sh

```diff
@@ -20,8 +20,8 @@ fi
 #in mac os 'cp' cannot create destination dir, so create it first
 mkdir ${LOCAL_ASSEMBLY_DIR}
 
-echo "Copying ${base_dir}/../../che-theia-init-sources.yml --> ${LOCAL_ASSEMBLY_DIR}/che-theia-init-sources.yml"
-cp "${base_dir}/../../che-theia-init-sources.yml" "${LOCAL_ASSEMBLY_DIR}/"
+echo "Compresing 'che-theia' --> ${LOCAL_ASSEMBLY_DIR}/che-theia.tar.gz"
+git ls-files -z -c -o --exclude-standard | xargs -0 tar rvf ${LOCAL_ASSEMBLY_DIR}/che-theia.tar.gz
```

Comment:
hello, why do we need to use git/tar instead of adding the folder ?

Reply:
We cannot copy files outside of docker context:
```
The <src> path must be inside the context of the build; you cannot COPY ../something /something, because the first step of a docker build is to send the context directory (and subdirectories) to the docker daemon.
```

Reply:
can't we just copy the files to the docker context ?

Reply:
We can, but I think tar/untar may be faster

Reply:
I think there is something odd as when I tried from theia folder, it zipped only local files (from current folder)

Reply:
yes, I fix that, now should work fine